android/src/com/mapswithme/util/UiUtils.java

```diff
@@ -360,6 +361,20 @@ public static boolean isViewTouched(@NonNull MotionEvent event, @NonNull View vi
     return viewRect.contains(x, y);
   }
 
+  public static void setFullscreen(@NonNull Activity activity, boolean fullscreen)
```

Comment:
There is more and more code that should be manually called in many situations for the app to work properly. Is there any way to avoid it and somehow code in the resources, for example?

Reply:
If the app was properly separated in Fragments, we could define a style for each where we could set the system bars visibility and other. But in current master Fragments are not used as often as they should so it is not possible.

Reply:
Let's plan this refactoring for the future )